app/lib/definitions.ts

```diff
@@ -19,8 +19,10 @@ export type carouselSettings = {
 
 //This type is based on the info in our database, at some point we should fix so we don't have both Screenings and Screening as types if it doesn't fill a purpose
 export type Screening = {
+  _id: number;
   movie: string;
   saloon: string;
   date: string;
   runtime?: string;
+  poster: string;
```

Comment:
I don't think "poster" is necessarily a property on all screening objects, for example it will not be used when presenting specific screenings on the /filmer/[id] page, so I think it should have a "?" like runtime

Reply:
Good note! Ive changed it now.